src/main/java/no/nav/pto/veilarbportefolje/postgres/PostgresAktivitetMapper.java

```diff
@@ -117,6 +118,19 @@ private static void byggAktivitetBrukerData(AvtaltAktivitetEntity avtaltAktivite
                 .setForrigeAktivitetStart(forrigeAktivitetStart.map(DateUtils::toIsoUTC).orElse(null));
     }
 
+    private static void byggStillingFraNavData(List<AktivitetEntityDto> aktiviteter, Set<String> aktiveAktiviteter, AktivitetEntity entity){
+        if(aktiveAktiviteter.contains(stilling_fra_nav.name())){
+            LocalDate yesterday = LocalDate.now().minusDays(1);
+            Optional<AktivitetEntityDto> nesteStillingFraNav = aktiviteter.stream()
+                    .filter(aktivitetEntityDto -> stilling_fra_nav.equals(aktivitetEntityDto.aktivitetsType))
+                    .filter(aktivitetEntityDto -> aktivitetEntityDto.svarfristStillingFraNav != null)
+                    .filter(aktivitetEntityDto -> LocalDate.parse(aktivitetEntityDto.svarfristStillingFraNav.substring(0,10)).isAfter(yesterday))
+                    .min(Comparator.comparing(frist -> frist.svarfristStillingFraNav));
+            entity.setNesteCvKanDelesStatus(nesteStillingFraNav.map(AktivitetEntityDto::getCvKanDelesStatus).orElse(null));
+            entity.setNesteSvarfristStillingFraNav(nesteStillingFraNav.map((AktivitetEntityDto::getSvarfristStillingFraNav)).orElse(null));
+        }
+    }
+
```

Comment:
Alle feltene i AktivitetStatusBrukerData er datoer, stilling fra nav er en type med to felt inni seg, så vi valgte å legge det i en egen funksjon siden det skilte seg litt fra de som tidligere var lagt inn her